### ISSUE TITLE
How many Epochs Does it Take to Reproduct Results?

### ISSUE THREAD

**Issue by riven314** [czyssrs/Few-Shot-NLG#10]
I follow your instructions stated in README to train the model on "humans" dataset. I found that by default ```Main.py``` would train the model for 5000 epochs and each epoch takes quite some time. 
May I ask does it really take 5000 epochs in order to reproduce the results stated in your paper?

**Comment by riven314** [comment]
btw, below are the metrics I got thus far at epoch 237:
```
540 : loss = 514.970, copyloss = 499.673, covloss = 20.444
copy BLEU: 28.3731
```
The BLEU is pretty close to the figure reported in your paper (29.5 in Table 1). But the ```copyloss``` seems a bit high (no reference for me to compare). Is it expected?

**Comment by riven314** [comment]
From what I saw, the model converges in a few hundred epoch. After the epoch 237, the loss and BLEU roughly stay the same
